TemplateApplication/Firestore/FirestoreConfiguration.swift

```diff
@@ -0,0 +1,94 @@
+//
+// This source file is part of the Stanford Spezi Template Application open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseFirestore
+import FirebaseStorage
+import Spezi
+import SpeziAccount
+import SpeziFirebaseAccount
+
+
+final class FireConfiguration: Module, DefaultInitializable, @unchecked Sendable {
```

Comment:
We should probably name this Firebase to ensure that this is not confused with HL7 FHIR.
```suggestion
final class FirebaseConfiguration: Module, DefaultInitializable, @unchecked Sendable {
```

Would also make sense to rename the file accordingly.